src/main/java/de/tum/in/www1/artemis/web/rest/TextExerciseResource.java

```diff
@@ -127,7 +134,11 @@ public TextExerciseResource(TextExerciseRepository textExerciseRepository, TextE
 
         TextExercise result = textExerciseRepository.save(textExercise);
         textClusteringScheduleService.ifPresent(service -> service.scheduleExerciseForClusteringIfRequired(result));
-        groupNotificationService.notifyTutorGroupAboutExerciseCreated(textExercise);
+
+        // If an exerciseGroup is set, we assume that the exercise is created for an exam. Then don't notify tutors when the exercise is created
+        if (!textExercise.hasExerciseGroup()) {
```

Comment:
```suggestion
        if (textExercise.hasCourse()) {
```

I think this check is easier and would allow the comment above to be much shorter